docs/components/interactive-map.md

```diff
@@ -0,0 +1,121 @@
+# InteractiveMap
+
+A stateful `React` component.
+This wraps `StaticMap` and provide additional built-in interactivity handling.
```

Comment:
I think this is also a secondary observation. The first comment should be that this component renders a mapbox-gl map and provides full event handling support.